tests/installation/grub_test.pm

```diff
@@ -0,0 +1,44 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+use strict;
+use base "basetest";
+use testapi;
+
+sub run() {
+    my $self = shift;
+    up_and_down();
+    assert_screen "grub2";
+    send_key 'esc';    # prevent grub2 timeout
```

Comment:
it's look like that isn't my machine that has problem.
Here:
https://openqa.suse.de/tests/197318/modules/livecdreboot/steps/2/src
and :  https://openqa.suse.de/tests/197318/modules/livecdreboot/steps/1/src
. and there are so many example, with the stuff  with send key up doesn't work.

If you want that i change the code, i don't mind, but we still have other issues, because situation happens that we have slowly performance. ( i tried with my machine running 3 workers).

So  i know you don't like the dirty aspect of my solution, but i don't like the tests failures.
So before something will stable, ( in the api or backend, that's your focus) i will propose to take this workaround, to avoid such situations, that aren't my machine issues.